src/main/java/lotto/model/Statistics.java

```diff
@@ -6,32 +6,42 @@
 
 public class Statistics {
     private Map<Rank, Integer> resultMap = new HashMap() {{
-        put(Rank.getRank(6), 0);
-        put(Rank.getRank(5), 0);
-        put(Rank.getRank(4), 0);
-        put(Rank.getRank(3), 0);
-        put(Rank.getRank(0), 0);
+        put(Rank.valueOf(6, false), 0);
+        put(Rank.valueOf(5, true), 0);
+        put(Rank.valueOf(5, false), 0);
+        put(Rank.valueOf(4, false), 0);
+        put(Rank.valueOf(3, false), 0);
+        put(Rank.valueOf(0, false), 0);
     }};
 
     public Map<Rank, Integer> getResultMap() {
         return resultMap;
     }
 
-    public Statistics(Lotto win, List<Lotto> lottos) {
-        lottos.forEach(lotto -> compareNumber(win, lotto));
+    public Statistics(Lotto win, LottoNumber bonus, List<Lotto> lottos) {
+        validateBonus(win, bonus);
+        lottos.forEach(lotto -> compareNumber(win, lotto, bonus));
     }
 
-    private void compareNumber(Lotto win, Lotto lotto) {
+    private void validateBonus(Lotto win, LottoNumber bonus) {
+        if(win.getLottoNumber().contains(bonus)) {
```

Comment:
```suggestion
        if (win.getLottoNumber().contains(bonus)) {
```

매의 눈 @_@

IntelliJ Auto Formatting 기능을 사용하는 습관을 들여보아요.

The auto-formatting shortcut in IntelliJ

* For windows Ctrl + Alt + L.
* For ubuntu Ctrl + Alt + windows + L.
* For Mac Command + Option + L.

Reply:
네~ 쓰고 있는데 중간에 자꾸 누락시켰네요 ㅜㅜ 

IntelliJ에서 저장하면  단축키없이 자동으로 포맷팅되는 방법이 있을까요?
아니면 prettier 설정으로 커밋할때 처리되게 해야 할까요?

Reply:
https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_module_directory 해당 링크를 참고해주세요 :)